tests/terraform/parser/resources/parser_scenarios/ternaries/main.tf

```diff
@@ -6,10 +6,9 @@ locals {
   bool_true  = true ? "correct" : "wrong"
   bool_false = false ? "wrong" : "correct"
 
-//  local_true = true
-  // TODO: HCL2 parser doesn't like the following line
-//  multiline = (local.local_true) ?
-//    "correct" : "wrong"
+  local_true = true
+  multiline = (local.local_true) ?
```

Comment:
Thanks @arielkru now the parser is not breaking!
@robeden I think the method `_process_vars_and_locals` is removing the `multiline` local from the dictionary.  So it's missing currently at the expected value.

Reply:
nvm. found the issue.